Packs/Zscaler/Integrations/Zscaler/Zscaler.yml

```diff
@@ -1883,7 +1883,7 @@ script:
       predefined:
       - "True"
       - "False"
-      description: Boolean value for user to choose, if wanted to get liter version of command which will retrive ID,Name,Type.
+      description: Boolean value for user to choose, To retrive limited version of command(ID,Name,Type).
```

Comment:
Let's rephrase the description here to,
`To retrieve only limited information of IP Destination groups. Includes ID, Name and Type of the IP Destination groups.`